runtime/peaq/src/lib.rs

```diff
@@ -756,19 +756,19 @@ pub mod staking {
 			pub const MinRequiredCollators: u32 = 4;
 			/// We only allow one delegation per round.
 			pub const MaxDelegationsPerRound: u32 = 1;
-			/// Maximum 25 delegators per collator at launch, might be increased later
+			/// No maximum number of delegators per collator at launch
 			#[derive(Debug, PartialEq, Eq)]
-			pub const MaxDelegatorsPerCollator: u32 = 25;
+			pub const MaxDelegatorsPerCollator: u32 = u32::MAX;
```

Comment:
Please keep this as 25 because it depends on the old/new token economy limitation now (Just the same as Krest's setting now)